main.py

```diff
@@ -1,7 +1,32 @@
+import asyncio
+import sys
+import msvcrt
+
 height = 40 #40 characters
 width = 172 #171 characters
```

Comment:
check to see if we can use this for height and width: https://www.geeksforgeeks.org/python-os-get_terminal_size-method/#:~:text=os.get_terminal_size%20%28%29%20method%20in%20Python%20is%20used%20to,represents%20height%20of%20the%20terminal%20window%20in%20characters.

Reply:
implemented this function